app/src/main/cpp/skyline/services/fssrv/IFileSystemProxy.cpp

```diff
@@ -61,16 +63,21 @@ namespace skyline::service::fssrv {
         }();
 
         manager.RegisterService(std::make_shared<IFileSystem>(std::make_shared<vfs::OsFileSystem>(state.os->appFilesPath + "/switch" + saveDataPath), state, manager), session, response);
+        return {};
+
     }
 
-    void IFileSystemProxy::OpenDataStorageByCurrentProcess(type::KSession &session, ipc::IpcRequest &request, ipc::IpcResponse &response) {
+    Result IFileSystemProxy::OpenDataStorageByCurrentProcess(type::KSession &session, ipc::IpcRequest &request, ipc::IpcResponse &response) {
         if (state.loader->romFs)
             manager.RegisterService(std::make_shared<IStorage>(state.loader->romFs, state, manager), session, response);
         else
             throw exception("Tried to call OpenDataStorageByCurrentProcess without a valid RomFS");
```

Comment:
Is there no appropriate error code for this?

Reply:
Also not part of the pr :)